doc/contribute/contributor_expectations.rst

```diff
@@ -305,10 +305,23 @@ Reviewer Expectations
   they address all non-blocking comments. PR authors should acknowledge every
   review comment in some way, even if it's just with an emoticon.
 
+- Style changes that the reviewer disagrees with but that are not documented as
+  part of the project can be pointed out as non-blocking, but cannot constitute
+  a reason for a request for changes. The reviewer can optionally correct any
+  potential inconsistencies in the tree, document the new guidelines or rules,
+  and then enforce them as part of the review.
+
+- Whenever requesting changes, reviewers must be able to point out the
+  corresponding guideline, rule or rationale in the project's documentation.
+  This does evidently not apply to certain types of requests for changes,
```

Comment:
"evidently" is a strange choice of word in such a formal document. If there if evidence it should make an explicit reference to it.